packages/my-react-package/src/ContractWithWallet.tsx

```diff
@@ -132,14 +151,22 @@ class ContractWithWallet {
      * @param {ArgsJSON[]} argsJSON - Arguments of the function to execute
      * @returns 
      */
-    async sendSignedTransaction(signedTx:string) {
+    async sendSignedTx(
```

Comment:
Can you take another look through the code and make the functions/attributes private when the user won't/shouldn't directly use them?